public/style/base/_reset.scss

```diff
@@ -0,0 +1,78 @@
+html {
+  // Prevent adjustments of font size after orientation changes in IE and iOS. */
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+a,
+a:visited {
+  border-bottom: 1px dotted $brand-primary;
+  color: $brand-primary;
+  text-decoration: none;
+  // tasteful descenders
+  text-shadow: -1px -1px 0 $text-shadow, // left top
+        1px -1px 0 $text-shadow, // right top
+        -1px 1px 0 $text-shadow, // left bottom
+        1px 1px 0 $text-shadow, // right bottom
+        0 2px 0 $text-shadow; // bottom bottom
+  transition: all linear 125ms;
+  transition-property: box-shadow, color, border-bottom;
+ }
+
+ a:hover,
+ a:active,
+ a:focus {
+  border-bottom: 1px solid $brand-primary-darker;
+  // box-shadow: 0px 1px 1px red;
+  color: $brand-primary-darker;
+}
+
+html,
+body,
+div,
+span,
+article,
+header,
+footer,
+ol,
+ul,
+li,
+p {
+  border: 0;
+  font-size: 100%;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+}
+
+// HTML5 display-role reset for older browsers
+article,
+aside,
+footer,
+header,
+nav,
+section {
+  display: block;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+body {
+  background: $gray-lighter;
+  // Default system fonts in different operating systems
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
Probably best to move the font stack out to typography for sure. Background should maybe live elsewhere?